timber-lint/src/test/java/timber/lint/WrongTimberUsageDetectorTest.kt

```diff
@@ -0,0 +1,829 @@
+package timber.lint
+
+import com.android.tools.lint.checks.infrastructure.TestFiles.java
+import com.android.tools.lint.checks.infrastructure.TestFiles.manifest
+import com.android.tools.lint.checks.infrastructure.TestLintTask.lint
+import org.junit.Test
+
+class WrongTimberUsageDetectorTest {
+  private val TIMBER_STUB = java("""
+      |package timber.log;
+      |public class Timber {
+      |  public static void d(String s, Object... args) {}
+      |  public static void d(Throwable t, String s, Object... args) {}
+      |  public static Tree tag(String tag) { return new Tree(); }
+      |  public static class Tree {
+      |    public void d(String s, Object... args) {}
+      |    public void d(Throwable t, String s, Object... args) {}
+      |  }
+      |  private static final Tree TREE_OF_SOULS = new Tree();
+      |}""".trimMargin())
+
+  @Test fun usingAndroidLogWithTwoArguments() {
+    lint() //
```

Comment:
I'd drop the `//` here or are they still needed regarding formatting?

Reply:
If the current version of `square/java-code-styles` doesn't change the newline or indentation between `lint()` and `.files()`, then it should be OK to remove.